org.gnupg.GPA.yaml

```diff
@@ -0,0 +1,54 @@
+app-id: org.gnupg.GPA
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: gpa
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  #- --filesystem=~/.gnupg:rw
+  #- --filesystem=xdg-run/gnupg:rw
+  - --filesystem=host
+  - --filesystem=/tmp
+
+modules:
+
+  - shared-modules/gtk2/gtk2.json
+
+  - name: gpgme
+    sources:
+      - type: archive
+        url: https://www.gnupg.org/ftp/gcrypt/gpgme/gpgme-1.16.0.tar.bz2
+        sha256: 6c8cc4aedb10d5d4c905894ba1d850544619ee765606ac43df7405865de29ed0
+
+  - name: gpa
+    cleanup:
+      - "*.a"
+      - "*.la"
+      - "/cache"
+      - "/include"
+      - "/lib/*.a"
+      - "/lib/pkgconfig"
+      - "/man"
+      - "/share/aclocal"
+      - "/share/devhelp"
+      - "/share/gtk-doc"
+      - "/share/man"
+      - "/share/pkgconfig"
+    buildsystem: simple
+    sources:
```

Comment:
usually sources are last. it doesn't matter to flatpak-builder but it improves readability